designs.js

```diff
@@ -2,15 +2,21 @@ const listTitle = document.getElementById('inputTitle');
 const listQa= document.getElementById('inputQa');
 const listOfItems = document.getElementById('listBook');
 const submit = document.getElementById('MakeCatalog');
-  submit.addEventListener('submit', function (e){
-    e.preventDefault();
-    makeList();
-  }
-)
+submit.addEventListener('submit', makeList);
+clearForm();
```

Comment:
These naming looks inconsistent. Some buttons are buttonAction, some actionButton. Could you please fix this. I would also suggest to rename html element ids respectively.